examples/kubernetes/connectivity-check/main_tool.cue

```diff
@@ -15,14 +15,15 @@ objectSets: [
 	ingressCNP,
 ]
 
-globalFlags: "[-t component=<component>] [-t kind=<kind>] [-t name=<name>] [-t topology=<topology>] [-t quarantine=true]"
+globalFlags: "[-t component=<component>] [-t kind=<kind>] [-t name=<name>] [-t topology=<topology>] [-t quarantine=true] [-t type=<tooltype>]"
 
 ccCommand: {
 	#flags: {
-		component:  "all" | *"default" | "network" | "policy" | "services" | "hostport" | "proxy" @tag(component,short=all|default|network|policy|services|hostport|proxy)
+		component:  "all" | *"default" | "dns" | "network" | "policy" | "services" | "hostport" | "proxy" @tag(component,short=all|default|dns|network|policy|services|hostport|proxy)
```

Comment:
This hunk is actually no longer necessary, I decided to put the new tool under "component: proxy" instead.